### PR TITLE
buildroot: Move to 2016.08 release

### PR DESCRIPTION
**Pull request by shenki**
Package changes:

 - ethtool: bump to version 4.6
 - dropbear: bump version to 2016.74 (severe security bugfixes)
 - ncurses: various packaging fixes
 - i2c-tools: packaging changes
 - gcc: bump 4.9 series to 4.9.4
 - busybox: bump to version 1.25.0
 - elfutils: bump version to 0.166
 - glibc: bump default to version 2.23
 - eudev: bump version to 3.2. libgudev has moved to it's own package
 - kmod: bump version to 23
 - lvm2: bump version to 2.02.162
 - pciutils: bump to version 3.5.1

Configuration changes:

 - BR2_GLIBC_VERSION_2_23=y is now the default defconfigs that specified
   it do not require it set anymore.

 - The mambo and firenze defconfigs were regenerated to remove redundant
   information. The resulting .config should have no differences.

Signed-off-by: Joel Stanley <joel@jms.id.au>

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/615)

### CODE REVIEW

**Comment by shenki** [comment]
These conflicted as @ploetzma's p9 patches have gone in. I rebased but have not done a built of the new commit.

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "buildroot"]
 	path = buildroot
-	branch = 2016.05-op-build
-	url = https://github.com/open-power/buildroot
+	branch = 2016.08-op-build
+	url = https://github.com/shenki/buildroot
```
Comment (stewart-ibm) [inline .gitmodules]: Should be https://github.com/open-power/buildroot rather than shenki :)

**Comment by stewart-ibm** [comment]
Apart from the buildroot submodule URL, looks good!

**Comment by shenki** [comment]
Thanks, fixed

File: openpower/configs/firenze_defconfig [hunk 2]
```diff
@@ -5,26 +5,11 @@ BR2_OPENPOWER_POWER8=y
...
-BR2_TOOLCHAIN_BUILDROOT_CXX=y
-BR2_GLIBC_VERSION_2_23=y
-
-# No HostBoot, OCC, or PNOR required
-BR2_PACKAGE_HOSTBOOT=n
-BR2_PACKAGE_HOSTBOOT_BINARIES=n
 
 BR2_OPENPOWER_CONFIG_NAME="openpower-firenze"
 
-BR2_PACKAGE_OCC=n
```
Comment (williamspatrick) [inline openpower/configs/firenze_defconfig]: Were these all purposefully removed from your Mambo configs?

**Comment by williamspatrick** [comment]
Need witherspoon added now.

**Comment by shenki** [comment]
Fixed witherspoon.

The OCC change was intentional

**Pull request merged**